workflow/models.py

```diff
@@ -402,6 +402,7 @@ def __str__(self):
 # other programs is 'Program Dashboard'
 class FundCode(models.Model):
     name = models.CharField('Fund Code', max_length=255, blank=True)
+    funding = models.IntegerField('Funding', blank=True)
```

Comment:
Can you make the name be `percentage_of_funding` instead of `funding`? Also, for the label can we use `% of Funding`?

Reply:
Will do!